src/arrays/ArrayUtil.java

```diff
@@ -36,60 +69,99 @@ public static void create2dArray(int x, int y) {
      * @return true if the array contains the given value, otherwise return false.
      */
     public static boolean contains(int[] array, int value) {
-
+        for(int i=0;i<array.length-1;i++) {
+            if (array[i] == value) {
+                return true;
+            }
+        }
         return false;
     }
 
+
     /**
      * @param array An array to search in.
      * @param value The value to search for.
      * @return the first index of value in the array, if array does not contain given element return -1.
      */
     public static int indexOf(int[] array, int value) {
```

Comment:
Method does not work correctly, method can not get index of last element in array